packages/rs-drive-proof-verifier/src/types.rs

```diff
@@ -48,6 +49,10 @@ pub type IdentityBalance = u64;
 /// Identity balance and revision of the identity.
 pub type IdentityBalanceAndRevision = (u64, Revision);
 
+/// An identity contract nonce
+#[derive(Debug)]
+pub struct IdentityContractNonceFetcher(pub IdentityContractNonce);
```

Comment:
I don't like the name, it sounds like it's some active thing, while the idea is that this should be data model.

Reply:
I think it's not needed at all,

Reply:
@lklimek no, because it conflicts with the u64 of the balance.